query/test_duckdb.ipynb

```diff
@@ -0,0 +1,1648 @@
+    "\n",
+    "Here is another example with a 10x bigger dataset, i.e,. ~80K Patients and ~17M Observations. This\n",
+    "query takes over **23 minutes** with DuckDB, but only **20 seconds** with Spark (on the same machine)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "id": "c29fae1a-a2af-46d2-a4da-ba186b03f057",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2024-05-15T18:11:46.346523Z",
+     "iopub.status.busy": "2024-05-15T18:11:46.346251Z",
+     "iopub.status.idle": "2024-05-15T18:35:13.256216Z",
+     "shell.execute_reply": "2024-05-15T18:35:13.255196Z",
+     "shell.execute_reply.started": "2024-05-15T18:11:46.346504Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "15d36c38d4d24e0abb44fdb0045a9c94",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "FloatProgress(value=0.0, layout=Layout(width='auto'), style=ProgressStyle(bar_color='black'))"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>count_star()</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>597553</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   count_star()\n",
+       "0        597553"
+      ]
+     },
+     "execution_count": 45,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "con.execute(\"\"\"\n",
+    "SELECT COUNT(*) FROM(\n",
+    "SELECT\n",
+    "  O.id AS obs_id, OCC.system, OCC.code, O.status AS status,\n",
+    "  O.subject.PatientId AS patient_id, OVCC.code AS val_code\n",
+    "FROM '/dwh/OUT_from-json_791562/Observation/*.parquet' AS O,\n",
+    "  LATERAL (SELECT unnest(O.code.coding) AS OCC),\n",
+    "  LATERAL (SELECT unnest(O.value.codeableConcept.coding) AS OVCC)\n",
+    "WHERE OCC.code LIKE '1255%'\n",
+    "  AND OVCC.code LIKE '1256%'\n",
+    "); \"\"\").df()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3054a86-3249-42ec-baf9-46acbeae9fc5",
+   "metadata": {},
+   "source": [
+    "### CREATE VIEW\n",
```

Comment:
Do we want to add any comments on the performance numbers for Views over here?

Reply:
I suppose you mean doing aggregate queries on the `VIEW` and measure their performance. I don't think that will add much value on top of what we have already shown above; I mean that should show the same/similar behavior when compared to Spark.